api/docker/awslambda/buildspec.yml

```diff
@@ -0,0 +1,26 @@
+version: 0.2
+
+# Variables to set
```

Comment:
minor - would be nice to add checks for required parameters

Reply:
done